include/vara/Feature/Location.h

```diff
@@ -0,0 +1,69 @@
+#ifndef VARA_FEATURE_LOCATION_H
+#define VARA_FEATURE_LOCATION_H
+
+#if defined(STD_EXPERIMENTAL_FILESYSTEM)
+#warning "using std::experimental::filesystem"
```

Comment:
```suggestion
```
I don't think we need a warning in this case. It's not like the code is wrong or something, it's just that the user has an compiler that is not new "enough" to directly support this.